src/sinks/loki.rs

```diff
@@ -88,7 +103,9 @@ impl SinkConfig for LokiConfig {
             return Err("`labels` must include at least one label.".into());
         }
 
-        let request_settings = self.request.unwrap_with(&TowerRequestConfig::default());
+        let mut request_settings = self.request.unwrap_with(&TowerRequestConfig::default());
+        request_settings.concurrency = Some(1);
```

Comment:
It would probably be worth showing a warning if this was set to a value larger than 1.

Reply:
Instead of hard-coding to 1, maybe we could still let users set it if they really know what they are doing, but output a warning?

Reply:
True, but we are adding the ability to rewrite timestamps here so if they don't care about exact ordering they could achieve better throughput with higher concurrency.

I did open #6041 for better partitioning of requests by streams (labels) that would allow for concurrency.

Reply:
To clarify, a PR for #6041 would only revert `request.concurency = 1`, we would still need to track latest_timestamp of each stream and drop or rewrite timestamp when necessary.